src/routes/middlewares/emailVerification.js

```diff
@@ -0,0 +1,50 @@
+import { Op } from 'sequelize';
+import models from '../../db/models';
```

Comment:
we can import `User`  directly from models like
```javascript 
import { User } from ' ../../db/models'
```

Reply:
We agreed on using this convention because named exports was not done by sequelize